packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -520,16 +533,30 @@ export class IamPolicyChecksWebview extends VueWebview {
             } else {
                 jsonOutput.BlockingFindings.forEach((finding: any) => {
                     this.pushCustomCheckDiagnostic(diagnostics, finding, true)
+                    if (finding.findingType === 'ERROR') {
+                        if (
+                            finding.message.includes(
+                                'The policy in existingPolicyDocument is invalid. Principal is a prohibited policy element.'
```

Comment:
This will not be valid for CheckAccessNotGranted if this method is leveraged for both the checks (CheckNoNewAccess & CheckAccessNotGranted)

Reply:
If not being re-used for CheckAccessNotGranted, can we make the method name indicate it?

Reply:
not a launch blocker, but let's clean it up before launch preferably.

Reply:
Yes it will not be valid for CheckAccessNotGranted, I created a helper method to indicate that. The response is inherent to only CheckNoNewAccess, so it should not overlap.